pytds/tds.py

```diff
@@ -3180,6 +3180,8 @@ def make_param(self, name, value):
         if value is default:
             column.flags |= fDefaultValue
             value = None
+            if value_type is _Default:
```

Comment:
this seems like redundant test, value is already an instance of _Default hence value_type is _Default, no?